src/wavesurfer.ts

```diff
@@ -76,6 +76,8 @@ export type WaveSurferOptions = {
   fetchParams?: RequestInit
   /** Playback "backend" to use, defaults to MediaElement */
   backend?: 'WebAudio' | 'MediaElement'
+  /** Nonce for CSP if necessary */
+  nonce?: string
```

Comment:
Should we call it cspNonce maybe?

Reply:
Possibly, I was just following the pattern I saw other libs using, which is this one, but cspNonce would be more explict.

Reply:
Done